### PR TITLE
doc associations

### PR DESCRIPTION
**Pull request by opyate**
Backend `get_current_user` to only decode JWT and not fetch user from DB as this starved the DB/pool of connections.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -56,8 +56,6 @@ SECRET_KEY=super_secret
 # machine user account for loader
 MACHINE_USER_LOADER_EMAIL=loader
 MACHINE_USER_LOADER_PASSWORD=password
-# machine user token has no expiry
-MACHINE_USER_LOADER_JWT=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJsb2FkZXIiLCJwZXJtaXNzaW9ucyI6ImFkbWluIn0.GRymxg-mbw2oO2zEBQm8p8eS7MqUSvBolljWgTufEcI
```
Comment (opyate) [inline .env.example:60]: Machine user now just logs in before POSTing to backend.

File: backend/app/core/auth.py [hunk 2]
```diff
@@ -8,16 +8,18 @@
 from app.db import session
 from app.db.crud.user import get_user_by_email
 from app.db.models import User
+from app.db.schemas.user import JWTUser
```
Comment (opyate) [inline backend/app/core/auth.py:11]: The main change here is that there are now 2 ways to inject a user dependency in the API endpoints:
- get_current_active_user 
- get_current_active_db_user
(and their superuser analogues)

The former is quick (only a JWT decode) and the latter is heavy (fetches user from DB).
This sped up the doc upload quite a bit, as it doesn't need the DB user, but the request is kept open for a long time as Boto uploads PDFs to S3. This starved the DB pool with the original dependency.

File: loader/app/db/schema.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class AssociationSchema:
+    """A remote association, to be posted to backend.
+
+    TODO move to common schemas.
```
Comment (opyate) [inline loader/app/db/schema.py:8]: Maybe one day we can move all schemas to common. But now right now :)

File: loader/app/loaders/loader_cclw_v2/load/main.py [hunk 4]
```diff
@@ -62,6 +62,13 @@ async def load(ctx: Context, policies: PolicyLookup):
     logger.info(f"Done, imported {doc_count} docs from {len(policies.items())} actions")
 
 
+def warmup_local_caches():
```
Comment (opyate) [inline loader/app/loaders/loader_cclw_v2/load/main.py:65]: As some of the loader POSTs/uploads are now called from futures, best to warm up the cache outside of a possible stampeding herd.

File: loader/app/db/schema.py (hunk 3 above)
Comment (opyate) [reply loader/app/db/schema.py:8]: Also, then change this from a dataclass to a pydantic model.

**Review by joel-wright — APPROVED** [review]
LGTM

**Pull request merged**